main/2021/WG-07-14.md

```diff
@@ -46,4 +46,139 @@ None.
 
 ## Meeting Notes
 
-To be added after the meeting.
+### Attendees
+Luke Wagner
+
+Derek Schuff
+
+Deepti Gandluri
+
+Eric Prud’Hommeaux
+
+Conrad Watt
+
+Ryan Hunt
+
+Sergey Rubanov
+
+### Proposals and discussions
+
+#### SIMD to Phase 5
+
+DG: we’ve been testing implementations and tools, no issues have come up with implementations or users, so we thought it would be a good time to poll for phase 5. Should we do a consensus, or more detailed poll?
+
+LW: I expect we can get consensus. Ryan, do yo have any comments from the Firefox side?
+
+RH: I’m not actually a WG member but i’m here to answer questions about FF if anyone has them. We don’t have issues with our implementation, and we’re supportive of moving to phase 5.
+
+DG: any objections then?
+
+<no objections noted>
+
+Consensus. Feature moves to phase 5
+
+#### Publish First Public Working Draft, and auto-publishing
+EP: For our purposes here, “TR” means w3.org TR; it means a doc that conforms to publication rules.
+The previous doc (the rec) is big and unwieldy. I decided to try to write a sphinx to TR tool to convert it automatically: https://github.com/w3c/sphinx-to-tr example output is at http://uu3.org/checkouts/w3c/sphinx-to-tr/out/
```

Comment:
may as well remove the example output line 'cause uu3.org points to my laptop when it's in a DMZ and when i get around to updating the record at gandi.